vnext/Desktop.IntegrationTests/HttpOriginPolicyIntegrationTest.cpp

```diff
@@ -743,6 +743,46 @@ TEST_CLASS(HttpOriginPolicyIntegrationTest)
     TestOriginPolicy(serverArgs, clientArgs, s_shouldFail);
   } // FullCors304ForSimpleGetFails
 
+  TEST_METHOD(OfficeDev_OfficeJS_4144)
+  {
+    SetRuntimeOptionString("Http.GlobalOrigin", "http://orig.in");
```

Comment:
Why this has hardcoded "orig.in" url?

Reply:
No strict reason. Just chose an URL that would suggest its intended purpose.
For these type of tests, only syntactic validation is done. These `Origin` URLs are never connected to.

Suggestions for less confusing test URLs are welcome.